pkg/windows/calico.go

```diff
@@ -330,8 +330,9 @@ func checkIfNetworkExists(n string) bool {
 	return true
 }
 
-func createExternalNetwork(backend string) error {
+func createExternalNetwork(backend string, vxlanAdapter string) error {
 	logrus.Debug("Creating external network")
+	logrus.Infof("TESTING - This is the value of vxlanAdapter: %s", vxlanAdapter)
```

Comment:
Is this still a WIP PR?

Reply:
Yes, I need the requester of this feature to test it because I don't have the env to do it. That's why I added this log :P